### DISCUSSION TITLE
Deriving an improved marker for freq magnitude plots

### DISCUSSION THREAD

**Discussion by csrabak** [ra3xdh/qucs_s#261] [Ideas]
Given the machinery is "almost there" as the present marker has the ability to edit in the properties the specific frequency of interest (instead of the boring click and check, polluting the graph with several until the desired is found and them delete the unwanted ones... [kudos for the enhancement!]), I suggest we see a means¹ to be able to have the hp  and lp -3dB points found in the graph.

I foresee that depending upon the number of points selected for the simulation, the 'exact' frequency could not be found, and as healthy ensuing discussion in this thread we can find what would be the most useful: say finding the nearest point, interpolate, etc.?

I think this would be an useful addition to improve the ergonomics of qucs-s!

[1] I've to concede this is not an original idea, another commercial Spice simulator implements this.

**Comment by ra3xdh** [comment]
Yes, this feature is obvious for markers. I was also considering to add something like finding peak/valley values. These functions could be added to the right click menu of the marker. Automatic recognition of the -3dB points on the magnitude response may be harder to implement. 

> instead of the boring click and check, polluting the graph with several until the desired is found and them delete the unwanted ones

You don't need to remove and add marker again. You can move the marker with arrows keys (left and right) on keyboard and set the exact position. The up/down arrows works for parametric sweep diagrams.

**Reply by csrabak** [reply]
Thanks a lot for this tip!
